performance/multi-node/random_access.c

```diff
@@ -348,8 +323,8 @@ static inline void exchange_registration_data(fabric_t *fabric)
 }
 
 /* function declarations */
-CTPM_SIMPLE_TYPE_FUNC(int_sum_all, int, +);
-CTPM_SIMPLE_TYPE_FUNC(int64_sum_all, int64_t, +);
+CT_ALLREDUCE_FUNC(int_sum_all, int, +);
```

Comment:
`_all` at the end of these names seems redundant with the renaming of the function to include `allreduce`, unless the second *all* is means something else.